content/docs/redhat/acm/submariner/rosa/rosa-submariner.md

```diff
@@ -0,0 +1,706 @@
+spec:
+  channel: release-2.7
+  installPlanApproval: Automatic
+  name: advanced-cluster-management
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+```
+
+* Check that the Operator has installed successfully
+
+```sh
+oc get csv
+NAME                                 DISPLAY                                      VERSION   REPLACES   PHASE
+advanced-cluster-management.v2.7.0   Advanced Cluster Management for Kubernetes   2.7.0                Succeeded
+```
+
+NOTE: ACM Submariner will only work from 2.7 onwards! Ensure that you have a >= 2.7 ACM version.
+
+* Install MultiClusterHub instance in the ACM namespace
+
+```sh
+cat << EOF | kubectl apply -f -
+apiVersion: operator.open-cluster-management.io/v1
+kind: MultiClusterHub
+metadata:
+  namespace: open-cluster-management
+  name: multiclusterhub
+spec: {}
+EOF
+```
+
+* Check that the MultiClusterHub is properly installed
+
+```sh
+kubectl get mch -n open-cluster-management multiclusterhub -o jsonpath='{.status.phase}'
+```
+
+NOTE: if it's not in Running state, wait a couple of minutes and check again.
+
+## Deploy First ROSA Cluster
+
+* Define the prerequisites for install the ROSA cluster
+
+```sh
+ export VERSION=4.11.36 \
+        ROSA_CLUSTER_NAME_1=rosa-sbmr1 \
+        AWS_ACCOUNT_ID=`aws sts get-caller-identity --query Account --output text` \
+        REGION=eu-west-1 \
+        AWS_PAGER="" \
+        CIDR="10.0.0.0/16"
+```
+
+NOTE: it's critical that the Machine CIDR of the ROSA clusters not overlap, for that reason we're setting different CIDRs than the out of the box ROSA cluster install.  
+
+* Create the IAM Account Roles
+
+```sh
+rosa create account-roles --mode auto --yes
+```
+
+* Generate a STS ROSA cluster
+
+```sh
+rosa create cluster -y --cluster-name ${ROSA_CLUSTER_NAME_1} \
+--region ${REGION} --version ${VERSION} \
+--machine-cidr $CIDR \
+--sts
+```
+
+* Create the Operator and OIDC Roles
+
+```sh
+rosa create operator-roles --cluster ${ROSA_CLUSTER_NAME_1} --mode auto --yes
+rosa create oidc-provider --cluster ${ROSA_CLUSTER_NAME_1} --mode auto --yes
+```
+
+* Check the status of the Rosa cluster (40 mins wait until is in ready status)
+
+```sh
+rosa describe cluster --cluster ${ROSA_CLUSTER_NAME_1} | grep State
+State:                      ready
+```
+
+* Set the admin user for the ROSA cluster
+
+```sh
+rosa create admin --cluster=$ROSA_CLUSTER_NAME_1
+```
+
+* Login into the rosa cluster and set the proper context
+
+```sh
+oc login https://api.rosa-sbmr1.xxx.xxx.xxx.com:6443 --username cluster-admin --password xxx
+
+kubectl config rename-context $(oc config current-context) $ROSA_CLUSTER_NAME_1
+kubectl config use $ROSA_CLUSTER_NAME_1
+
+kubectl get dns cluster -o jsonpath='{.spec.baseDomain}'
+```
+
+### Generate ROSA New nodes for submariner 
+
+* Create new node/s that will be used to run Submariner gateway using the following command (check https://github.com/submariner-io/submariner/issues/1896 for more details)
```

Comment:
Turn the GitHub issue into a link here please.

Reply:
ack.